src/cmd/login/login.go

```diff
@@ -68,25 +69,59 @@ func login(_ *cobra.Command, _ []string) error {
 			return err
 		}
 		prints.PrintCliStderr("User registered as Otterize user with user ID: %s", user.Id)
-		userId = user.Id
+		meResponse, err = c.MeQueryWithResponse(registerCtxTimeout)
+		if err != nil {
+			return err
+		}
 	} else if err != nil {
 		return err
-	} else {
-		userId = meResponse.JSON200.User.Id
 	}
 
-	// query user to get full user info
-	userResponse, err := c.UserQueryWithResponse(registerCtxTimeout, userId)
-	if err != nil {
-		return err
-	}
-
-	user := lo.FromPtr(userResponse.JSON200)
+	// FIXME: create task to
+	user := meResponse.JSON200.User
 	prints.PrintCliStderr("Logged in as Otterize user %s (%s)", user.Id, user.Email)
-	if user.Organization != nil && user.Organization.Id != "" {
-		prints.PrintCliStderr("User is registered with organization %s", user.Organization.Id)
+	if len(*user.Organizations) != 0 {
+		prints.PrintCliStderr("You belong to the following organizations:")
+		for _, org := range *user.Organizations {
+			if org.Name != nil {
+				prints.PrintCliStderr("ID: %s | Name: %s", org.Id, *org.Name)
+			} else {
+				prints.PrintCliStderr("ID: %s", org.Id)
+			}
+		}
+		if len(*user.Organizations) == 1 {
+			prints.PrintCliStderr("Only 1 organization found - auto-selecting this organization for use. To change, join another organization and log in again.")
+		} else {
+			prints.PrintCliStderr("More than 1 organization found, input org ID (to change, log-in again, blank to select first org): ")
+			reader := bufio.NewReader(os.Stdin)
+			orgId, err := reader.ReadString('\n')
+			if err != nil {
+				return err
+			}
+
+			if strings.TrimSpace(orgId) == "" {
+				orgId = (*user.Organizations)[0].Id
+			}
+
+			err = config.SaveSelectedOrganization(config.OrganizationConfig{OrganizationId: orgId})
+			if err != nil {
+				return err
+			}
+		}
+		prints.PrintCliStderr("User is registered with organization %s", viper.GetString(config.ApiSelectedOrganizationId))
 	} else {
-		prints.PrintCliStderr("User is not registered with any organization, please use the Otterize CLI or UI to create or join an organization")
+		prints.PrintCliStderr("User has no organization, automatically creating one for you.")
```

Comment:
There's a reason we didn't auto-create one here, and that's to best simulate the UI behaviour, and to remove logic from the CLI. 
Users who haven't joined any organizations yet is a valid use case, and what they should do next is either create a new org (using `otterize organizations create`) or list & accept an invite (using `otterize invites list / accept`). 
This goes back to my previous comment - we should support users logging in via CLI without / before selecting an org, and allow selecting later on.

Reply:
Removed auto org creation, now fails on log-in with no orgs